p2panda-rs/src/document/document_id.rs

```diff
@@ -23,7 +25,7 @@ use crate::{Human, Validate};
 ///                         \
 ///                          \__ [UPDATE] (Hash: "eff..")
 /// ```
-#[derive(Clone, Debug, StdHash, Ord, PartialOrd, Eq, PartialEq)]
+#[derive(Clone, Debug, StdHash, Ord, PartialOrd, Eq, PartialEq, Serialize, Deserialize)]
```

Comment:
We can't derive `Deserialize` as this introduces unchecked values. You can implement `Deserialize` yourself and validate the String, see `Hash` or `DocumentViewId` for examples :+1:

Reply:
ah, ok... Thanks!

Reply:
And maybe you missed my comment in the other PR but maybe double check if we need serde for scalars ✌️